### PR TITLE
Added typing speed test

### PR DESCRIPTION
**Pull request by abhishek-iiit**
# Description

Added Typing speed test

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [ x ] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [ x ] My code follows the style guidelines(Clean Code) of this project
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas
- [ x ] I have created a helpful and easy to understand `README.md`
- [ x ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [ x ] My changes generate no new warnings
- [ x ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by abhishek-iiit** [comment]
Made within SWOC

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Add a Issue for the Same

File: typing speed test/linter.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (atarax665) [inline typing speed test/linter.yml:1]: What is the use of super linter in typing speed test project?

File: typing speed test/main.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        self.active = False
+        self.input_text = ''
+        self.word = ''
+        self.results = 'Time:0 Accuracy:0 % WPM:0 '
+        self.start_time = 0
+        self.overall_time = 0
+        self.accuracy = '0%'
+
+        pygame.init()
+        self.image_open = pygame.image.load(os.path.abspath("/home/abhishek/Desktop/GITHUB/SWOC/Amazing-Python-Scripts/typing speed test/typing_speed.png"))
```
Comment (atarax665) [inline typing speed test/main.py:28]: The image path should be edited accordingly when the project is cloned on the local machine. Please do mention this here as in-code comment and also mention it in the README.

File: typing speed test/readme.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Package/Script Name
+Typing Speed Test
+
+## Setup instructions
+
+before running the scipt,
```
Comment (atarax665) [inline typing speed test/readme.md:6]: Include all the steps from cloning this repo to running the main.py file.

File: typing speed test/readme.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+## Detailed explanation of script, if needed
+Typing is all about muscle memory, so the only way to improve is to practice typing regularly. 
+
+You should always start typing by placing your fingers on the home row. There are the small bumps on the F and J key, which indicate the initial position of your index fingers. It helps you find this starting position on the keyboard without looking at it. Once you start with this position, your fingers have the full range of motion and a proper distance to all keys. It is the most optimal hands placement during typing.
+
+* The WPM stands for words per minute, and it is a measure of typing speed, commonly used in the recruitment process and typing speed tests.
+
+## Output
+
+Attached in the file named 'screenshot.png'
```
Comment (atarax665) [inline typing speed test/readme.md:19]: Add output image link here.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**